notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Perhaps some text can be added here to build some intuition. ie - If I am just looking at the notebook and not listening to you talk it would still be nice to have a few bullet points motivating what grid search is and why we introduce it here.   One thing to think about - in intro we build some motivation around what estimators are etc. Might be worth describing that grid search is another form of estimator and is in fact a forecaster/why this is a good choice.  However, I could also see this being potentially confusing? Worth thinking about what the balance should be! :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I aggree with Mirae, it would be helpful to do some motivation at the beginning of this part. Perhaps, you can wrap up the previous topics and then motivate why pipelining and grid search is beneficial. This would allow the listeners to comeback if they were lost in the previous parts and it also allows them to refocus on the new topic. </p>